challenge_4/sass/main.scss

```
@@ -0,0 +1,60 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+$primary-color: #998235;
+$secondary-color: #0B2429;
+
+.container {
+  background: $secondary-color;
+  height: 300px;
+  width: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 60px;
+}
+
+.left-triangle {
+  height: 55px;
+  border-top: 50px solid transparent;
+  border-bottom: 50px solid transparent;
+  border-left: 48px solid $primary-color;
+}
+
+.right-triangle {
+  height: 55px;
+    border-top: 50px solid transparent;
+    border-bottom: 50px solid transparent;
+    border-right: 48px solid #998235;
+    margin-right: -14px;
+}
```

Both of these triangles share most of the properties, so you are repeating styles with the same values. To avoid this, you could add a class to both triangles called `triangle`, then inside that class goes all the styles shared between them and in the `.right-triangle` and `.left-triangle` classes the specific style for each triangle. And also, since you're now using sass, you can use the nesting superpower and it should be something like this: 

```suggestion
.triangle {
  // common styles for both triangles
  
  &-right {
    // styles for the right triangle
  }
  
  &-left {
    // styles for the left triangle
  }
}
```
Keep in mind that if you want to nest the styles like this, the classes for the triangles should be named `triangle-right` and `triangle-left`.